src/unfold/contrib/filters/admin.py

```diff
@@ -44,17 +58,21 @@ def __init__(self, field, request, params, model, model_admin, field_path):
             value = params.pop(self.parameter_name)
             self.used_parameters[self.parameter_name] = value
 
-    def queryset(self, request, queryset):
+    def queryset(
+        self, request: HttpRequest, queryset: QuerySet[Any]
+    ) -> Union[None, QuerySet]:
         if self.value():
             return queryset.filter(**{self.parameter_name: self.value()})
 
-    def value(self):
+    def value(self) -> Any:
         return self.used_parameters.get(self.parameter_name, None)
 
-    def expected_parameters(self):
+    def expected_parameters(self) -> List[Union[None, str]]:
         return [self.parameter_name]
 
-    def choices(self, changelist):
+    def choices(
+        self, changelist: ChangeList
+    ) -> Tuple[Dict[str, Any],]:
```

Comment:
`Tuple[Dict[str, Any], ...]` with three dots it specifies there can be any amount of elements inside tuple returned